Fix QA issues related to implicit declarations.

* QA Notice: Package triggers severe warnings which indicate that it
*            may exhibit random runtime failures.
* clmadjust.c:184:10: warning: implicit declaration of function ‘mcxFree’ [-Wimplicit-function-declaration]
* mcxi.c:42:16: warning: implicit declaration of function ‘isatty’ [-Wimplicit-function-declaration]

--- a/src/impala/iface.h
+++ b/src/impala/iface.h
@@ -16,15 +16,15 @@
 
 /* This file currently provides nothing */
 
-dim nu_meet_can  ;
-dim nu_meet_sl   ;
-dim nu_meet_zip  ;
-dim nu_diff_can  ;
-dim nu_diff_sl   ;
-dim nu_diff_zip  ;
-double nu_magic  ;
+extern dim nu_meet_can  ;
+extern dim nu_meet_sl   ;
+extern dim nu_meet_zip  ;
+extern dim nu_diff_can  ;
+extern dim nu_diff_sl   ;
+extern dim nu_diff_zip  ;
+extern double nu_magic  ;
 
-dim mclx_n_thread_g  ;
+extern dim mclx_n_thread_g  ;
 
 #endif
 
--- a/src/shcl/clmadjust.c
+++ b/src/shcl/clmadjust.c
@@ -14,6 +14,7 @@
 #include "report.h"
 #include "clmadjust.h"
 
+#include "util/alloc.h"
 #include "util/io.h"
 #include "util/types.h"
 #include "util/err.h"
--- a/src/shmcx/mcxi.c
+++ b/src/shmcx/mcxi.c
@@ -8,6 +8,7 @@
 */
 
 #include <stdio.h>
+#include <unistd.h>
 
 #include "stack.h"
 #include "glob.h"
